### DISCUSSION TITLE
Reverse proxy issue

### DISCUSSION THREAD

**Discussion by ChrisL79** [magicbug/Cloudlog#1781] [Q&A]
My instance is not working behind a reverse proxy. It was for an hour or so, then it stopped. Going to my .com address that points to my cloudlog install generates a 502 bad gateway.

Is there something that needs to be changed in a config file? If so, which one?

Cheers!

**Comment by magicbug** [comment]
One server got an invalid response from another so first port of call will be the server logs to see what its complaining about.

**Reply by ChrisL79** [reply]
That issue is fixed. I was using an Android phone to configure the proxy and mistakingly added a . where one was not needed.
The only remaining issue now, is where i enter my .com name in the address bar from outaide my network, it resolves to the clouldlog instance, but the address changes to my lan address. 

I think this may be a php config error. Can you point me in the right direction maybe?

**Reply by ChrisL79** [reply]
I am hosting this on ESXI if that helps.
